app/src/main/res/values/strings.xml

```diff
@@ -282,7 +282,6 @@
 
   <string name="share_app_title">Share App</string>
   <string name="share_coordinates_not_present">Coordinates were not specified during image selection</string>
-  <string name="something_went_wrong">Something went wrong</string>
-  <string name="slow_internet">You seem to have slow internet connection</string>
+  <string name="error_fetching_nearby_places">Error fetching nearby places.</string>
```

Comment:
How about changing the message to "Error fetching nearby places **that need images**" thus mentioning what *Nearby* should actually be used for, as stated in [my comment previously](20d3c378d4c3a58ae1d7b1d50064d43b9000a093#r187400433))? What *Nearby* should be used for isn't mentioned in a lot of places. So, it might be helpful to the user who see the error message.

Reply:
IMO, If it's not mentioned at other places then just mentioning it in the error messages won't be that useful. After #706 this should be a bit more clear. :)